### DISCUSSION TITLE
Template card with badge for Icon

### DISCUSSION THREAD

**Discussion by Disane87** [piitaya/lovelace-mushroom#870] [Ideas]
It would be great to have the opportunity to set an icon badge for the template card (or all others) to indicate some actions or states in the icon, like the person card has for the "at home" state. 

![image](https://user-images.githubusercontent.com/956158/203361106-a649956f-5f2d-41cf-83f1-3a213c50826d.jpeg)

My use case:

I have a Twitch integration in HA and want to indicate the state of the Twitch channel (like online) in the icon itself, so the card won't be big as hell:

![image](https://user-images.githubusercontent.com/956158/203361729-a74a01c6-26db-4878-8dd2-1d0509129519.jpeg)

**Comment by piitaya** [comment]
It's already possible with the template card 😊

**Comment by Disane87** [comment]
> It's already possible with the template card 😊

@piitaya Oh sorry! Had the doc of the template card opened already all the day but never saw this. Sorry for bothering you with this dumb request 🫤
